app/views/admin/instances/index.html.haml

```diff
@@ -15,10 +15,14 @@
   .filter-subset
     %strong= t('admin.instances.moderation.title')
     %ul
-      %li= filter_link_to t('admin.instances.moderation.all'), limited: nil
-
-      - unless limited_federation_mode?
-        %li= filter_link_to t('admin.instances.moderation.limited'), limited: '1'
+      - if limited_federation_mode?
+        %li= filter_link_to t('admin.instances.moderation.allowed'), status: nil
```

Comment:
In limited federation mode, I think it's still possible to know about instances but not currently be allowing federation with them (e.g., previously federated with and the removed the domain allow)? Should we support surfacing those instances?

Reply:
It is possible for records to persist for a short amount of time, yes, but accounts are all immediately marked as suspended on deletion of a `DomainAllow` and scheduled for immediate deletion.

This means that outside of these transitional moments, limited federation mode is expected to only know about allowed servers.

This is also why you should run `tootctl domains purge --limited-federation-mode` when switching to limited federation mode.

Reply:
Alrighty